.github/workflows/package-rpms.yml

```diff
@@ -0,0 +1,99 @@
+name: Package BCDA, Worker RPMs
+
+on:
+  workflow_call:
+    inputs:
+      release_version:
+        description: 'Release version (or branch name)'
+        required: true
+        type: string
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Release version (or branch name)'
+        required: true
+        type: string
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  package_bcda_rpms:
+    name: Package BCDA, Worker RPMs
+    runs-on: self-hosted
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v3
+        with:
+          aws-region: ${{ vars.AWS_REGION }}
+          role-to-assume: arn:aws:iam::${{ secrets.ACCOUNT_ID }}:role/delegatedadmin/developer/bcda-prod-github-actions
```

Comment:
Personal preference but I would move the assume-role down as far as it can go, right before the step where it's first needed.